File: api/src/main/java/net/blueshell/api/repository/survey/AnswerRepository.java
package net.blueshell.api.repository.survey;

import net.blueshell.api.base.BaseRepository;
import net.blueshell.api.model.survey.Answer;
import org.springframework.stereotype.Repository;

import java.util.Set;

@Repository
public interface AnswerRepository extends BaseRepository<Answer> {
    Set<Answer> findByQuestionSurveyId(Long surveyId);
}

File: api/src/main/kotlin/net/blueshell/api/repository/survey/AnswerRepository.kt
package net.blueshell.api.repository.survey

import net.blueshell.api.model.survey.Answer
import net.blueshell.api.repository.base.BaseRepository
import org.springframework.stereotype.Repository

@Suppress("FunctionName")
@Repository
interface AnswerRepository : BaseRepository<Answer, Long> {
    fun findBy_questionSurveyId(surveyId: Long): MutableSet<Answer>
}